packages/core/src/feed.ts

```diff
@@ -0,0 +1,58 @@
+import { Subject, Observable, map, NextObserver } from 'rxjs'
+import { type StreamMetadata, type StreamState } from '@ceramicnetwork/common'
+import { CommitID } from '@ceramicnetwork/streamid'
+import { StreamUtils } from '../../common/lib/utils/stream-utils.js' //TODO: once commitIdFromStreamState is part of the package remove this line, and update the @ceramicnetwork/common import
+import type { AnchorStatus } from '@ceramicnetwork/common'
+
+export type Document = {
+  commitId: CommitID
+  content: any
+  anchorStatus: AnchorStatus
```

Comment:
are we sure we need to include this at all?  This was still an open question in the design doc, we had a note to follow up with Orbis and see if they need this.  I'd prefer to err on the side of leaving this out unless it's specifically asked for.

I think if users care about information from anchoring, they're more likely to care about the anchor timestamp than the AnchorStatus anyway, so I'd like to wait and think about how we expose anchor-related information more holistically if and when it becomes a requirement.

Reply:
Looping in @rohhan for this question.  This corresponds to this open question from the Notion document: https://www.notion.so/Firehose-Requirements-and-API-Stub-1c8db095218d4e2db9b0ee1db9dce861?d=cb6573e840944a53b5f819be2292b6c1&pvs=4#16107f227a4a48f9b7fd216e98ff3a8c

Reply:
@JulissaDantes Per discussion, we should remove this field, adding later as needed.

Reply:
Ready!